diktat-test-framework/src/main/kotlin/org/cqfn/diktat/test/framework/processing/TestComparatorUnit.kt

```diff
@@ -21,10 +21,14 @@ class TestComparatorUnit(private val resourceFilePath: String,
     /**
      * @param expectedResult
      * @param testFileStr
+     * @param smoke
      * @return true if transformed file equals expected result, false otherwise
      */
     @Suppress("FUNCTION_BOOLEAN_PREFIX")
-    fun compareFilesFromResources(expectedResult: String, testFileStr: String): Boolean {
+    fun compareFilesFromResources(
+        expectedResult: String,
+        testFileStr: String,
+        smoke: Boolean = false): Boolean {
```

Comment:
```suggestion
        trimLastEmptyLine: Boolean = false): Boolean {
```
To make it look like a more general option

Reply:
Fixed